CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 This is a history of changes to clara-rules.
 
+# 0.23.0
```

Comment:
Minor: Perhaps change to 0.23.0-SNAPSHOT and then back to 0.23.0 when the release is cut so that anyone reading this isn't confused into thinking 0.23.0 has been released when it hasn't.  This probably doesn't matter if you're planning to release 0.23.0 upon merging this though.

Reply:
Will do since I might wait until I also merge #487 to do a release.